project_ux/views/project_project_views.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0"?>
```

Comment:
both view files are marked as executable files, please run a chmox -x file in order to fix it